packages/vite/src/node/plugins/css.ts

```diff
@@ -811,8 +811,8 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
 
         return css
       }
-      let extractedCss = extractCss()
-      if (extractedCss && !hasEmitted) {
+      let extractedCss = !hasEmitted && extractCss()
```

Comment:
By checking `hasEmitted` first, we can avoid unnecessary computation here. @sapphi-red What do you think?

Reply:
The change looks nice 👍